src/applications/personalization/search-representative/config/form.js

```diff
@@ -0,0 +1,68 @@
+// import fullSchema from 'vets-json-schema/dist/000-schema.json';
```

Comment:
Is this still relevant? If not, can we remove?

Reply:
Removed